### ISSUE TITLE
[Synergy] : New-OvLogicalInterconnectGroup with -UplinkSet Param : UplinkSet Name = Lig Name

### ISSUE THREAD

**Issue by odesaphy** [HewlettPackard/POSH-HPOneView#612]
My code : 

```
$uls_eth = New-OvUplinkSet -Name "uls01_ethernet" -Type Ethernet -NetworkSets (Get-OvNetworkSet -Name "my_networkset") -UplinkPorts "Enclosure1:Bay3:Q1", "Enclosure1:Bay3:Q2", "Enclosure2:Bay6:Q1", "Enclosure2:Bay6:Q2" -FecMode Cl91 -PassThru

New-OvLogicalInterconnectGroup -Name "lig36" -FrameCount 2 -InterconnectBaySet 3 -FabricModuleType "SEVC100F32" -DownlinkSpeedMode 50 -Bays @{Frame1 = @{Bay3 = 'SEVC100F32' ; Bay6 = 'SE50ILM'};Frame2 = @{Bay3 = 'SE50ILM'; Bay6 = 'SEVC100F32' }} -FabricRedundancy 'HighlyAvailable' -UplinkSet $uls_eth
```

LIG "lig36" is created with one Ethernet UplinkSet named "lig36" 

![image](https://user-images.githubusercontent.com/25363168/178735650-6b2af598-ed61-4177-85ec-e59f573c435e.png)

Tested with POSH 6.30.2928.3678 on a Synergy OV 6.60

The issue is in Function BuildUplinkSet on line 95278

`$_UplinkSetObject.name = $Name`
should be
`$_UplinkSetObject.name = $Params.Name`

**Comment by ChrisLynchHPE** [comment]
I just published an update to the 6.30 library that should address this issue.  Please update to [Release 6.30.3188.1900](https://github.com/HewlettPackard/POSH-HPEOneView/releases/tag/v6.30.3188.1900) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPEOneView.630/6.30.3188.1900).

Please verify before this issue is closed.

**Comment by odesaphy** [comment]
Issue is fixed. Thanks a lot.
